platforms/chibios/QMK_PROTON_C/convert_to_proton_c.mk

```diff
@@ -5,8 +5,8 @@ BOARD := QMK_PROTON_C
 OPT_DEFS += -DCONVERT_TO_PROTON_C
 
 # These are defaults based on what has been implemented for ARM boards
-AUDIO_ENABLE = yes
-WS2812_DRIVER = bitbang
+AUDIO_ENABLE ?= yes
+WS2812_DRIVER ?= bitbang
 
 # Force task driven PWM until ARM can provide automatic configuration
-BACKLIGHT_DRIVER = software
\ No newline at end of file
+BACKLIGHT_DRIVER ?= software
```

Comment:
This breaks converting an avr board that is currently using PWM backlight. The reason this sets software is as it needs zero config.

Reply:
In this case, wouldn't it be better to set the backlight driver defaults in `common_features.mk` based on the platform, then? 

Something like: 
```make
ifeq ($(PLATFORM),CHIBIOS)
    BACKLIGHT_DRIVER ?= software
else
    BACKLIGHT_DRIVER ?= pwm
endif
```

Reply:
The downside there, is it bleeds the CTPC implementation out. Right now, while maybe not the perfect behaviour, is at least isolated.

Reply:
In that case, I add this change then.